helm/hpcc/templates/_helpers.tpl

```diff
@@ -514,15 +520,11 @@ Check that the storage and spill planes for a component exist
 {{- define "hpcc.checkDefaultStoragePlane" -}}
 {{- if (hasKey .me "storagePlane") }}
  {{- $search := .me.storagePlane -}}
- {{- if ne (include "hpcc.isValidStoragePlane" (dict "search" $search "root" .root)) "true" -}}
-  {{- $_ := fail (printf "storage data plane %s for %s is not defined" $search .me.name ) }}
- {{- end -}}
+ {{- include "hpcc.isValidStoragePlane" (dict "search" $search "root" .root "category" "data" "type" "storage data" "for" .me.name) -}}
```

Comment:
minor, better to rename isValid to checkValid

Reply:
Alternative implementation would be to have a function that returns a list of all the storage planes, and include it in the fail message.